src/stores/game.js

```diff
@@ -512,7 +512,7 @@ export const useGameStore = defineStore('game', {
     // In-Game Moves //
     ///////////////////
     async requestDrawCard() {
-      await this.makeSocketRequest('draw');
+      await this.makeSocketRequest('draw', { moveType: 'draw' });
```

Comment:
I think maybe we should import the enum, this should probably have been done in points too

Reply:
Yes I think it deserves its own PR so that this can be a lean example of just what's needed for a move (once that's fixed)

Reply:
Resolved by #1026